new/rfcXXX-lifecycle-and-release-policies.md

```diff
@@ -0,0 +1,214 @@
+---
+RFC: unassigned
+Title: Chef OSS Lifecycle and Release Policies
+Author: Tom Duffield <tom@chef.io>
+Status: Draft
+Type: Process
+Replaces: RFC047, RFC081
+---
+
+# Chef OSS Lifecycle and Release Policies
+
+This RFC describes the lifecycle and release policies of Chef Open Source Software (OSS) that fall under the purview of the [Chef Maintainers](https://github.com/chef/chef/blob/master/MAINTAINERS.md).
+
+## Motivation
+
+    As a Chef maintainer,
+    I need an established versioning policy,
+    so I can version my software consistently.
```

Comment:
How about:
As a Chef user,
I need chef version numbers to give me expectations about the extent of chagnes
so I can estimate my upgrade efforts